languages/go/exercises/concept/basic-slices/.meta/design.md

```diff
@@ -0,0 +1,41 @@
+## Goal
+
+The goal of this exercise is to teach the student the basics on how to work with slices in Go.
+
+## Things to teach
+
+For example:
+
+- instantiate (len, cap)
+- append
+- set value
+- read value
+- remove value
+- taking length
+- subslices
+- expanding slices (e.g. in `append(slice1, slice2...)`)
+
+## Things not to teach
+
+For example:
+
+- inner workings of slices
+- arrays
+
+## Concepts
+
+- `basic-slices`
+
+## Prerequisites
+
+- iteration (which in turn depends on `numbers`, `functions`, `conditionals`)
```

Comment:
I think I'd list the individual prerequisites all here, as I'm interested in knowing what specifically from the `numbers` concept is required for this exercise.